templates/cluster-template-persistent-storage-kccm.yaml

```diff
@@ -64,11 +64,18 @@ spec:
                 memory:
                   guest: "4Gi"
                 devices:
+                  interfaces:
+                  - bridge: {}
+                    name: pod
+                    macAddress: 00:10:fa:6e:38:4a
```

Comment:
I don't think we can set static mac addresses in the template because the same mac will end up being used for all VMs stamped from this template.

Reply:
This is only to exercise CI with fixed mac's, finally we should use kubemacpool.